seleniumDescriptors.js

```diff
@@ -0,0 +1,177 @@
+var format = require('util').format;
```

Comment:
You're probably right. I should add tests around the configs to ensure their output doesn't change in the future.

Why wouldn't we split up modules that have different responsibilities?